src/openloco/input/mouse_input.cpp

```diff
@@ -307,6 +310,110 @@ namespace openloco::input
         }
     }
 
+    // 0x004C72ED
+    static void loc_4C72ED()
+    {
+        auto window = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
+        if (window == nullptr)
+            return;
+
+        constexpr uint16_t horizontalFlags = scroll_flags::HSCROLLBAR_THUMB_PRESSED | scroll_flags::HSCROLLBAR_LEFT_PRESSED | scroll_flags::HSCROLLBAR_RIGHT_PRESSED;
+        constexpr uint16_t verticalFlags = scroll_flags::VSCROLLBAR_THUMB_PRESSED | scroll_flags::VSCROLLBAR_UP_PRESSED | scroll_flags::VSCROLLBAR_DOWN_PRESSED;
+
+        window->scroll_areas[_currentScrollOffset / sizeof(ui::scroll_area_t)].flags &= ~(verticalFlags | horizontalFlags);
```

Comment:
This looks quite different from the assembly. Either way, I think the scroll_area index should be computed on its own line.